expenseTracker/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "expensetracker",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "node app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "axios": "^1.7.9",
+    "bcryptjs": "^2.4.3",
+    "dotenv": "^16.4.7",
+    "express": "^4.21.2",
+    "express-async-errors": "^3.1.1",
+    "jsonwebtoken": "^9.0.2",
+    "mongodb": "^6.13.0"
```

Comment:
You should have `mongoose` in your dependencies.

It could be in your `node_modules` folder, but it should still be in your `package.json`.

Do:
`npm install mongoose@latest`
in your `expenseTracker` directory to get it here.